pkgs/applications/misc/yambar/default.nix

```diff
@@ -18,6 +18,7 @@
 , wayland-protocols
 , waylandSupport ? false
```

Comment:
```suggestion
, waylandSupport ? true
```

Reply:
And please put the wayland packages in a conditional if possible

Reply:
Done.